tests/integration/it_roles_test.py

```diff
@@ -9,20 +9,44 @@ def test_list(client):
     assert client.get("/roles/").status_code == HTTPStatus.OK
 
 
-def test_create(client):
-    response = client.post(url_for("role.create"), data={
-        "name": "owner"
-    })
-    assert response.location.endswith(url_for('role.list_roles'))
-    assert Role.query.count() == 1
-    assert Role.query.get(1).name == "owner"
+def test_create(client, db_session):
+    assert Role.query.count() == 0
+    created = None
+    try:
+        response = client.post(url_for("role.create"), data={
+            "name": "owner"
+        })
+        assert response.location.endswith(url_for('role.list_roles'))
+        assert Role.query.count() == 1
+        created = Role.query.get(1)
+        assert created.name == "owner"
+    finally:
+        if created:
+            db_session.delete(created)
+            db_session.commit()
 
 
 def test_edit(client):
-    assert client.get("/roles/edit/1").status_code == HTTPStatus.OK
+    assert client.get(url_for('role.edit', id=1)).status_code == HTTPStatus.NOT_FOUND
+
+
+def test_delete_not_found(client):
+    assert Role.query.filter_by(id=1).count() == 0
+    result = client.post(url_for('role.delete', id=1))
```

Comment:
@ekondrashev can we inline this variable? It's only used once

Reply:
@amihaiemil done, thx